lib/network_utils.pm

```diff
@@ -37,7 +37,7 @@ sub setup_static_network {
     $args{ip} //= '10.0.2.15';
     $args{gw} //= testapi::host_ip();
     $args{silent} //= 0;
-    configure_static_dns(get_host_resolv_conf(), $args{silent});
+    configure_static_dns(get_host_resolv_conf(), undef, $args{silent});
```

Comment:
Just for the record why the previous PR was reverted, is because it was missing this change.